docs/content/en/docs/intro-klt.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Introduction to the Keptn Lifecycle Toolkit
+linktitle: Introduction to the Keptn Lifecycle Toolkit
+description: Understand the Keptn Lifecycle Toolkit
+weight: 05
+cascade:
+  github_subdir: "docs/content/en/docs"
+  path_base_for_github_subdir: "/content/en/docs-dev"
+---
+
+The Keptn Lifecycle Toolkit (KLT)  makes your deployments observable,
+brings application-awareness to your Kubernetes cluster,
+and helps you reliably deliver your application with:
+
+* Pre-Deployment Tasks: e.g. checking for dependant services,
+  checking if the cluster is ready for the deployment, etc.
+* Pre-Deployment Evaluations: e.g. evaluate metrics
+  before your application gets deployed (e.g. layout of the cluster)
+* Post-Deployment Tasks: e.g. trigger a test,
+  trigger a deployment to another cluster, etc.
+* Post-Deployment Evaluations: e.g. evaluate the deployment,
+  evaluate the test results, etc.
+
+All of these things can be executed on a workload or on an application level,
+whereby a Keptn application is a collection of multiple workloads.
+
+## Compare KLT and Keptn V1-LTS
+
+The Keptn Lifecycle Controller (KLT) is an incubating CNCF project
+whose design reflects lessons we learned while developing Keptn V1.
+KLT recognizes that tools such as Argo and Flux
+are very good at deploying applications
+so adds pre-deployment and post-deployment evaluations and actions.
+For many installations, this provides the functionality they need
+with much less complexity than the Keptn V1 project.
+
+Keptn V1 is a fully-incubated, LTS release
```

Comment:
I would remove this section, as every keptn variant has other strengths and IMHO both configurations could get complex

Reply:
I would like to preserve a mention of Keptn LTS that gives  a nod to the fact that it is currently useful in some cases.   We can leave it to the reader to decide whether they are interested in those cases, of course ;-). See what you think of my rewrite.